site/docs/components/switch/accessibility.mdx

```diff
@@ -12,28 +12,28 @@ data:
 ### Best practices
 
 - On focus, the screen reader should:
-  - Identify the switch
-  - Read out the text
-  - Express the state
+  - Identify the switch.
+  - Read out the text.
+  - Express the state.
 
 ### Keyboard interactions
 
 <KeyboardControls>
 <KeyboardControl keyOrCombos="Tab">
 
-- If focus is on the previous element in the tab order, move focus onto Switch.
-- If focus is on Switch, moves focus to the next focusable element in the tab order.
+- If focus is on the previous element in the tab order, Tab moves focus to the switch.
+- If focus is on switch, Tab moves focus to the next focusable element in the tab order.
 
 </KeyboardControl>
 <KeyboardControl keyOrCombos="Space">
 
-- Activate Switch to change state.
+- Space activates switch to change state.
```

Comment:
A wider comment we refer to checked as "state" here but elsewhere we use "checked" directly. I don't have a preference but we should be consistent